manifests/params.pp

```diff
@@ -20,6 +20,6 @@
   $log_levels = {}
   $enable_ha = false
   $ha_node_ips = []
-  $ha_node_index = ''
```

Comment:
I think you're not running ODL in HA, so this shouldn't break you, but just making sure you see it @trozet